index.html

```diff
@@ -293,6 +293,41 @@ <h4>Skills</h4>
           <hr />
         </div>
       </section>
+      <section class="bg">
+        <section class="contact-me">
+          <div class="text">
+            <h1>Contact me</h1>
+            <p>
+              If you have an application you are interested in developing, a
+              feature that you need built or a project that needs coding. I’d
+              love to help with it
+            </p>
+          </div>
+          <form action="https://formspree.io/f/xrgdqdyv" method="post">
+            <div class="input-container">
+              <label for="name">
+                <input name="name" id="name" type="text" placeholder="enter name"
+                maxlength="30"
+                required/>
+              </label>
+              <label for="email">
+                <input name="Email" id="email" type="email" placeholder="enter email"
+                required/>
+              </label>
+              <textarea
+                name="textarea"
+                id="textarea"
+                cols="30"
+                rows="10"
+                maxlength="500"
+                  placeholder="write your message here"
+                  required
+              ></textarea>
+              <button class="form-btn" type="submit">Get It Touch</button>
```

Comment:
Kindly add button interactions as shown in the Figma design. Also fix the typo here, it should be "Get in Touch" - there is a typo in the Figma design 🤗

![image](https://user-images.githubusercontent.com/73714615/192571065-6653ec1a-10ad-49f4-adf5-f360636a3fd6.png)